packages/core/test/scxml.test.ts

```diff
@@ -168,30 +168,30 @@ const testGroups = {
     // 'test210.txml', // sendidexpr not supported yet
     // 'test215.txml', // <invoke typeexpr="...">
     // 'test216.txml', // <invoke srcexpr="...">
-    // 'test220.txml', // done.invoke used as inexact event descriptor
+    'test220.txml',
     // 'test223.txml', // idlocation not implemented yet
-    // 'test224.txml', // done.invoke used as inexact event descriptor
+    // 'test224.txml', // <invoke idlocation="...">
     // 'test225.txml', // unique invokeids generated at invoke time
     // 'test226.txml', // <invoke src="...">
-    // 'test228.txml', // done.invoke used as inexact event descriptor + _event.invokeid not implemented yet
+    'test228.txml',
     'test229.txml',
-    // 'test230.txml', // done.invoke used as inexact event descriptor + SCXML _event properties not implemented yet
```

Comment:
I don't think this should be uncommented - just yet. This is a "manual" test. Those manual tests usually just require observing the logged output so it's quite easy for us to actually automate them. We can spy `console.log` and snapshot results - but even if you'd end up doing that as part of this PR this test still shouldn't be accounted as "passing" because the snapshotted logs would not match expectations.